ipv8/messaging/anonymization/community.py

```diff
@@ -887,6 +887,13 @@ def on_extended(self, source_address, data, _):
         circuit = self.circuits[circuit_id]
         self._ours_on_created_extended(circuit, payload)
 
+    def handle_raw_data(self, circuit, origin, data):
```

Comment:
This is just nitpicking, but according to Pylint callbacks should start with `on_` or `cb_` or end with `_cb`. Could you use `on_` so in the future we don't get annoying Pylint errors?

Reply:
⏫

Reply:
Will change it. This PR is still WIP btw, I want to make some more changes 👍